packages/dd-trace/src/appsec/sdk/noop.js

```diff
@@ -1,6 +1,16 @@
 'use strict'
 
+class NoopAppsecSdkV2 {
+  trackUserLoginSuccess () {}
+
+  trackUserLoginFailure () {}
+}
+
 class NoopAppsecSdk {
+  constructor () {
+    this.v2 = new NoopAppsecSdkV2()
+  }
```

Comment:
this is still so weird to have the appsecSdkV2 be INSIDE of appsecSdkV1. I'm not gonna block the PR for that but it doesn't radiate hapiness

Reply:
I really don't like the idea of having:

```javascript
tracer.appsec.trackUserLoginSuccessEvent(...)
tracer.appsec.trackUserLoginSuccessV2(...)
tracer.appsec.trackUserLoginFailureEvent(...)
tracer.appsec.trackUserLoginFailureV2(...)
```

imo is cleaner if we have:
```javascript
tracer.appsec.trackUserLoginSuccessEvent(...)
tracer.appsec.trackUserLoginFailureEvent(...)
tracer.appsec.v2.trackUserLoginSuccess(...)
tracer.appsec.v2.trackUserLoginFailure(...)
```

Reply:
Maybe, something like `eventTrackingV2` as namespace could be acceptable also:
```javascript
tracer.appsec.trackUserLoginSuccessEvent(...)
tracer.appsec.trackUserLoginFailureEvent(...)
tracer.appsec.eventTrackingV2.trackUserLoginSuccess(...)
tracer.appsec.eventTrackingV2.trackUserLoginFailure(...)
```

Reply:
what about `tracer.appsecV2.trackUserLoginSuccess()` ?

Reply:
I am suggesting these names, instead of appsecV2, also because this comment:

https://github.com/DataDog/dd-trace-js/pull/5333#discussion_r2001199730

> It's not really a v2 of the appsec sdk, it's a v2 only for those specific functions